connect/mirror-client/src/main/java/org/apache/kafka/connect/mirror/DefaultReplicationPolicy.java

```diff
@@ -43,6 +47,11 @@ public void configure(Map<String, ?> props) {
             log.info("Using custom remote topic separator: '{}'", separator);
             separatorPattern = Pattern.compile(Pattern.quote(separator));
         }
+
+        if (props.containsKey(INTERNAL_TOPIC_SEPARATOR_ENABLED_CONFIG)) {
+            log.info("Disable the usage of topic separator for internal topics");
+            isInternalTopicSeparatorEnabled = Boolean.valueOf(props.get(INTERNAL_TOPIC_SEPARATOR_ENABLED_CONFIG).toString());
```

Comment:
My IDE pointed out to me that we're doing unnecessary boxing here; we can use `Boolean::parseBoolean` instead:
```suggestion
            isInternalTopicSeparatorEnabled = Boolean.parseBoolean(props.get(INTERNAL_TOPIC_SEPARATOR_ENABLED_CONFIG).toString());
```